kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/IPAddress.java

```diff
@@ -46,20 +46,7 @@
      */
     public static IPAddress getByAddress(byte[] addr) throws UnknownHostException {
         IPAddress result = null;
-        java.net.InetAddress jnetAddr = null;
```

Comment:
According to this it should also work with IPv6: http://download.java.net/jdk7/archive/b123/docs/api/java/net/InetAddress.html#getByAddress(byte[])

But I could also revert this if you believe it is necessary.